tests/agent/concurrent_requests.py

```diff
@@ -105,7 +105,7 @@ def load_test(self):
         self.logger.debug("Starting tornado I/O loop")
         ioloop.IOLoop.instance().start()
 
-    def check_counts(self, it, max_wait=60, backoff=.5):
+    def check_counts(self, it, max_wait=60, backoff=1):
```

Comment:
increase the time between count document checks on Elasticsearch, it removes load from Elasticsearch and duplicates the time to wait for the documents to be load, now we wait 60s it is fair enough.